eclipse-scout-core/src/table/Table.ts

```diff
@@ -353,6 +353,7 @@ export class Table extends Widget implements TableModel {
     this._group();
```

Comment:
Remove _group here and do it only for Scout classic in TableAdapter, so it won't be done twice on init?